gk/main.c

```diff
@@ -415,6 +427,14 @@ gk_process_granted(struct flow_entry *fe, struct ipacket *packet)
 	return 0;
```

Comment:
This dropped comment should become part of the explanation of returning zero in the previous item of this review.